### DISCUSSION TITLE
How can i change IoU with deepocsort?

### DISCUSSION THREAD

**Discussion by fangfang0304** [mikel-brostrom/yolo_tracking#1245] [Q&A] [answered]
hello How can i change IoU with deepocsort?

**Comment by mikel-brostrom** [comment, marked as answer]
You can change this:

https://github.com/mikel-brostrom/yolo_tracking/blob/90ef6ef3c7602473a431facebdbd4ef1ac1b38d3/boxmot/trackers/deepocsort/deep_ocsort.py#L320

 to any of these:

https://github.com/mikel-brostrom/yolo_tracking/blob/90ef6ef3c7602473a431facebdbd4ef1ac1b38d3/boxmot/utils/iou.py#L166-L169

**Reply by fangfang0304** [reply]
thank u!!!
